src/main/scala/scalaz/stream/Process.scala

```diff
@@ -737,8 +753,8 @@ object Process extends ProcessInstances {
   /** The `Process` which emits no values and signals normal termination. */
   val halt: Process0[Nothing] = halt0
 
-  /** Alias for `halt`. */
-  def empty[F[_],O]: Process[F, O] = halt
+  /** Empty emit. Used to signal end of evaluation form `step` to give chance step to be interrupted */
+  val empty: Emit[Nothing] = Emit(Nil)
```

Comment:
@pchlupacek I was thinking more just making `empty = halt` again.  Sometimes it semantically makes more sense to write `Process.empty` rather than `Process.halt`, and it's nice to have both.  If you feel it's best to remove `empty` entirely though, I'm not strongly opposed.  I was just objecting to the fact that `empty` and `halt` are different in this PR.